elyra/metadata/handlers.py

```diff
@@ -74,11 +74,20 @@ def _validate_body(self):
         body = self.get_json_body()
 
         # Ensure name, schema_name and metadata fields exist.
```

Comment:
nit: we should probably remove 'name' from the comment.

EDIT: Actually, I suppose 'name' is still a required field - we just defer its "required-ness", so it's probably fine to leave that comment alone.

Reply:
good catch. i can remove it from comment since it is mentioned in the comment that immediately follows that section